cluster/manager/manager.go

```diff
@@ -746,6 +748,7 @@ func (c *ClusterManager) joinCluster(
 	selfNodeEntry.NonQuorumMember =
 		selfNodeEntry.Status == api.Status_STATUS_DECOMMISSION ||
 			!c.quorumMember()
+	c.selfNode.NonQuorumMember = selfNodeEntry.NonQuorumMember
```

Comment:
You probably checked it already but wondering if should set NonQuorumMember on line 597 in initNode() as well.

Reply:
`initNode()` is called before the listeners are initialized. The caller of the `initNode()` is [initListeners()](https://github.com/libopenstorage/openstorage/blob/master/cluster/manager/manager.go#L1216), which does things in the following order - 
```
initNode()   - create a node object
initNodeInCluster() - initializes all the listeners
quorumMember() - this calls QuorumMember() on the now initialized PX listener
```
In this PR we are setting the quorum member flag after the above steps. We set it in both the selfNode and the node entry that will get stored in the cluster database.